test/issues/issues-test.js

```diff
@@ -479,4 +481,16 @@ describe("issues", function () {
             assert.same(originalSetTimeout, global.setTimeout, "fakeTimers restored");
         });
     });
+
+    describe("#1840 - sinon.restore useFakeXMLHttpRequest", function () {
+
```

Comment:
MINOR: Remove empty lines around the test below.

Reply:
Empty line removed.